bin/bix

```diff
@@ -561,6 +562,13 @@ tag_push() {
     done
 }
 
+do_ex_int_test() {
+    export MIX_ENV=test
+    run_mix "do" deps.get, compile --warnings-as-errors >/dev/null
+    run_mix test --trace --warnings-as-errors --all-warnings --only cluster_test
```

Comment:
From what I remember, we might need to run the test in apps/verify to ensure that other applications aren't started.